report_thesis/src/sections/proposed_approach/model_selection.tex

```diff
@@ -0,0 +1,103 @@
+\subsection{Model and Preprocessing Selection}
+Having established the validation and testing procedures, as well as, the evaluation metrics in our experiments, this section will detail our approach to model and preprocessing selection.
+
+Choosing the right models and preprocessing techniques for \gls{libs} data analysis is a challenging task. 
+As the literature highlighted in Section~\ref{sec:related-work} suggests, a variety of models and preprocessing techniques that promise to be adept at handling data that exhibits high-dimensionality, multi-collinearity, and matrix effects.
+Additionally, different machine learning models perform better on some oxides than others. 
```

Comment:
Unclear where we have this claim from in the previous sentence, as additionally implies another, separate claim. But this sentence, as I understand it, builds on the lit. review.
```suggestion
The literature also indicates that different machine learning models perform better on some oxides than others.
```